src/use-position-fixed.ts

```diff
@@ -96,7 +96,9 @@ export function usePositionFixed({
     if (nested || !hasBeenOpened) return;
     // This is needed to force Safari toolbar to show **before** the drawer starts animating to prevent a gnarly shift from happening
     if (isOpen) {
-      setPositionFixed();
+      // avoid for standalone mode (PWA)
+      const isStandalone = window.matchMedia('(display-mode: standalone)').matches;
+      !isStandalone && setPositionFixed();
```

Comment:
I think this needs an early return.

I tracked down a bug I was having only on iOS PWAs to vaul release `0.9.1`, and this commit is most likely the culprit.

The content below my drawer is jumping up and down, and the set-timeout in this if statement is most likely why.
If we're no longer fixing the position for PWAs, we don't need to restore it either, right?

@emilkowalski @Cuzart 

```suggestion
      // avoid for standalone mode (PWA)
      const isStandalone = window.matchMedia('(display-mode: standalone)').matches;
      if (isStandalone) return;
      setPositionFixed();
```